src/main/java/teammates/ui/controller/InstructorFeedbackSubmissionEditPageAction.java

```diff
@@ -18,8 +18,8 @@ protected boolean isSpecificUserJoinedCourse() {
     @Override
     protected void verifyAccesibleForSpecificUser(FeedbackSessionAttributes session) {
         InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, account.googleId);
-        boolean creatorOnly = false;
-        gateKeeper.verifyAccessible(instructor, session, creatorOnly);
+        boolean isEnabledForCreatorOnly = false;
```

Comment:
Use `isCreatorOnly` for consistency with [InstructorFeedbackResultsPageAction.java](https://github.com/xpdavid/teammates/blob/1bf08cd6ad9ff8e83025289cfc73b7380d3ffc81/src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageAction.java#L38). Also, update the function definition in `GateKeeper.java`.